superbench/runner/ansible.py

```diff
@@ -34,9 +34,9 @@ def __init__(self, config):
             if inventory_file or inventory_list:
                 self._config['host_pattern'] = 'all'
                 inventory = InventoryManager(loader=DataLoader(), sources=inventory_file or f'{inventory_list},')
-                host_list = inventory.get_groups_dict()['all']
-                if len(host_list) > 0:
-                    self._config['cmdline'] = '--forks {}'.format(len(host_list))
+                self._host_list = inventory.get_hosts(pattern='all', order='sorted')
```

Comment:
Why `self._host_list`? Do we need to store it?

Reply:
we need to at least store the sorted first host in this init() method, which will be used in update_mpi_config() method.